docusaurus/docusaurus.config.js

```diff
@@ -41,6 +41,12 @@ const config = {
                     customCss: require.resolve('./src/css/custom.css'),
                 },
             }),
+            {
+                gtag: {
+                    trackingID: 'UA-156258629-2',
+                    anonymizeIP: true,
+                },
+            },
```

Comment:
I think we have too many curls 
```suggestion
                gtag: {
                    trackingID: 'UA-156258629-2',
                    anonymizeIP: true,
                },
```

running locally I see this error with your code as written 
```
[ERROR] Error: "presets[0]" must contain 2 items
```

give me a yell if I can help you debug this

Reply:
Fixed! Thanks for reviewing.

I run it locally and used a [Google Analytics Debugger](https://chrome.google.com/webstore/detail/google-analytics-debugger) to check it, seems to work.
![Screenshot from 2022-04-28 21-08-29](https://user-images.githubusercontent.com/2302748/165829414-e5dc21b8-10b4-4334-bd02-1533e720f36c.png)


Otherwise, this plugin is always inactive in development and only active in production to avoid polluting the analytics statistics. I will only be able to check that it sends data correctly to Google Analytics once we run it on production.